### DISCUSSION TITLE
How to properly use module localIdentName in Nuxt 3?

### DISCUSSION THREAD

**Discussion by radonirinamaminiaina** [nuxt/nuxt#16615] [Questions]
Hello,

I have the following config:

```
import { defineNuxtConfig } from "nuxt";

// https://v3.nuxtjs.org/api/configuration/nuxt.config
export default defineNuxtConfig({
  build: {
    loaders: {
      cssModules: {
        modules: {
          localIdentName: "[hash:base64:6]",
        },
      },
    },
  },
});

```

I want to override the default value which is [**[local]_[hash:base64:5]**](https://v3.nuxtjs.org/api/configuration/nuxt.config#localidentname) but the code above doesn't work.

Any help will be welcomed.

Thank you

**Comment by ghost** [comment]
the reason this doesn't work is because this overwrites css modules config when used as a webpack plugin. nuxt 3 by default uses vite instead of webpack, so if you use it you have to overwrite vite css modules config:
```
export default defineNuxtConfig({
  vite: {
    css: {
      modules: {
        generateScopedName: "[hash:base64:6]",
      },
    },
  },
});
```
![Screenshot from 2022-10-15 20-31-16](https://user-images.githubusercontent.com/102010617/196002728-448f8689-34e3-49b5-9a9f-93478b7d08b7.png)
